seatunnel-core/seatunnel-spark-starter/src/main/java/org/apache/seatunnel/core/starter/spark/SparkStarter.java

```diff
@@ -191,7 +191,7 @@ private List<Path> getPluginsJarDependencies() throws IOException {
             return stream
```

Comment:
The same suggestion as above. Move this method to `org.apache.seatunnel.common.config`